apps/envoy-gateway/data.yaml

```diff
@@ -0,0 +1,40 @@
+title: "Envoy Gateway"
+tags:
+  - Networking
+summary: Manage your Application and API traffic with Envoy Gateway.
+logo: "./assets/envoy-gateway_logo.svg"
```

Comment:
will a horizonal svg be better here ?
https://github.com/cncf/artwork/blob/main/examples/graduated.md#envoy-gateway-logos

Reply:
tbh I think circle/square shaped logos are more consistent across apps overview